### ISSUE TITLE
Intermittently can't see pipes when ventcrawling

### ISSUE THREAD

**Issue by msgerbs** [tgstation/tgstation#52948]
## Reproduction: 

- Get staff of changed into headslug
- Start ventcrawling around
- After a while, I started not being able to see pipes

 I got staff of changed into a headslug and started ventcrawling around. After a while I noticed that pipes would sometimes disappear. Mechanically, everything kept working, I just couldn't see where I was going. Leaving the vents and going back in caused them to return but they would disappear again a short while later.

![2020-08-14 21_30_49-Window](https://user-images.githubusercontent.com/1313921/90303763-64fc8780-de76-11ea-9135-2b7b81528d88.png)

**Comment by LemonInTheDark** [comment]
did they just flicker out of view?

**Comment by msgerbs** [comment]
No, it stayed like that as I moved around. It wasn't just flickering. One second I was going through pipes and the next I was wandering in a dark void.

**Comment by LemonInTheDark** [comment]
ahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh here we go again

**Comment by msgerbs** [comment]
I got mindswapped into a mouse (I have really bad luck!) and ventcrawled dozens of times and didn't see this issue come back.

**Comment by nightred** [comment]
Tried several things but unable to cause the effect.

**Comment by Sealed101** [comment]
can't reproduce locally

**Comment by LemonInTheDark** [comment]
may have been fixed by <https://github.com/tgstation/tgstation/pull/66709>

**Comment by projectkepler-ru** [comment]
cannot reproduce  still

**Comment by Sealed101** [comment]
:)

**Comment by LemonInTheDark** [comment]
I think I might have fixed it guys

**Comment by nightred** [comment]
Glad to have this fixed. Amazing job lemon
